default.nix

```diff
@@ -13,7 +13,7 @@ let
     hello = callPackage ./ciment/hello { };
 
     # Casa
-    casa = callPackage ./ciment/casa/4.7.2.nix { };
```

Comment:
I hope this does not break previous installations?

Reply:
No it shouldn't. If a package has already been installed, it's now in the nix store and it will be removed by the garbage collector only if no user profile has links to it.